OpenRA.Mods.Common/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -61,7 +61,13 @@ public override void Draw()
 			{
 				var a = worldRenderer.Viewport.WorldToViewPx(dragStart);
 				var b = worldRenderer.Viewport.WorldToViewPx(mousePos);
-				Game.Renderer.RgbaColorRenderer.DrawRect(a, b, 1, Color.White);
+				var color = Color.White;
+				if (modifiers.HasFlag(Modifiers.Alt) && !modifiers.HasFlag(Modifiers.Ctrl))
+					color = Color.Cyan;
```

Comment:
Can we please expose these as ChromeMetrics so mods are able to customise or disable them?

Reply:
Done.